modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -109,6 +110,7 @@ export default {
       deep: true,
       handler() {
         this.updateNextAndEmit();
+        this.$emit('fieldStateChange', this.fieldState);
```

Comment:
Why do we need both this and `input`?

Reply:
Whatever trouble i was having riding the `input` event back out doesn't apply anymore.
Removed `fieldStateChange` event and tacked it onto the schema's `this.next` for evaluation higher up.